src/core/handle.ml

```diff
@@ -223,7 +250,8 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
             (* Add a symbol corresponding to the proof, with a warning. *)
             out 3 "(symb) %s (admit)\n" x.elt;
             wrn cmd.pos "Proof admitted.";
-            Sig_state.add_symbol ss e Const x a impl None
+            let (ss, _) = Sig_state.add_symbol ss e Const x a impl None
+            in ss
```

Comment:
Here and below, you can simply write `fst (Sig_state.add_symbol ss e Const x a impl None)`.

Reply:
At the end of the case `P_theorem`, the couple returned is really (ss, ...)?
It should be the result of the function `finalize`, no?
I think there is something I don't understand here!